packages/firestore/test/integration/util/helpers.ts

```diff
@@ -105,6 +106,14 @@ function apiDescribeInternal(
   }
 }
 
+// TODO(b/131094514): Remove after clearPersistence() is updated in index.d.ts.
+export async function _clearPersistence(
```

Comment:
You can rename this to clearPersistence() if you want.  We basically only ever use `_` as a way to hide methods on our public APIs (like FirebaseFirestore).